microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -571,3 +572,7 @@ def verify_uio_binding(
                 "/dev/uio0 still exists after driver unload"
             ),
         )
+
+    def _force_dpdk_default_source(self, variables: Dict[str, Any]) -> None:
+        if not variables.get("dpdk_source"):
```

Comment:
It may need to write like `variables.get("dpdk_source", None):`, in case this variable doesn't exist in runbook.